README.md

```diff
@@ -124,7 +128,7 @@ You should be able to do the same in Mac or Windows but note that setting up a H
     [openbdre@sandbox ~]$
     ```
 
-* Download Maven from a mirror, unpack and add to the PATH.
+* Download Maven from a mirror, unpack and add to the `PATH`.
```

Comment:
Probably don't need a code style highlight here as the code highlight is done immediately below